test/functional/tests/io_class/test_io_class_occupancy_load.py

```diff
@@ -94,9 +95,7 @@ def test_ioclass_occuppancy_load(cache_line_size):
 
     with TestRun.step(f"Perform IO with size equal to cache size"):
         for io_class in io_classes:
-            run_io_dir(
-                f"{io_class.dir_path}/tmp_file", int((cache_size) / Unit.Blocks4096)
-            )
+            run_io_dir(f"{io_class.dir_path}/tmp_file", int(cache_size / Unit.Blocks4096))
```

Comment:
Don't you have to `.get_value()` from `Unit.Blocks4096` here as well?

Reply:
Right. IDE missed it since its division of two variables with the same type. Any way, I have refactor this file again to unify the usage of Unit.Blocks4096 : )